pkg/tsdb/azuremonitor/loganalytics/azure-log-analytics-datasource.go

```diff
@@ -289,7 +289,9 @@ func (e *AzureLogAnalyticsDatasource) executeQuery(ctx context.Context, query *A
 
 	defer func() {
 		err := res.Body.Close()
-		backend.Logger.Error("Failed to close response body", "err", err)
+		if err != nil {
+			logger.Warn("failed to close response body", "error", err)
```

Comment:
I'm fine with the additional check but do we want change it to a warning, why? also why it's not capitalised?